gpu/kinfu_large_scale/tools/kinfuLS_app.cpp

```diff
@@ -1333,13 +1333,14 @@ main (int argc, char* argv[])
   //    return std::cout << std::endl << "Kinfu is supported only for Fermi and Kepler arhitectures. It is not even compiled for pre-Fermi by default. Exiting..." << std::endl, 1;
 
   pcl::shared_ptr<pcl::Grabber> capture;
-  bool triggered_capture = false;
-  bool pcd_input = false;
-  bool oni2_interface = false;
 
   std::string eval_folder, match_file, openni_device, oni_file, pcd_dir, oni2_file{};
   try
   {
+    bool triggered_capture = false;
```

Comment:
I believe this change will blow up. `triggered_capture` is still used in line 1441, `pcd_input` in line 1426 and `oni2_inerface` in 1448.

Reply:
We need the GPU compilation to catch such simple errors. Damn.

Nice catch @SergioRAgostinho

Reply:
Yeah completely missed it 😅
Fixed it now